feat: Show alert if scheduler is inactive or app version is stale

marination opened the pull request:
Resolves https://github.com/alyf-de/banking/issues/70

<img width="1301" alt="Screenshot 2023-11-02 at 4 10 07 PM" src="https://github.com/alyf-de/banking/assets/25857446/137c8184-f8d7-4eeb-a276-aea949f322db">

- The scheduler being inactive is a good enough check. If the scheduler is not working due to some kind of error, the web server would also most likely go down, taking down the site
- The app version check does a basic latest release vs in-app version comparison

marination committed:
feat: Show alert if scheduler is inactive or app version is stale

marination committed:
Merge branch 'version-14' into app-health

barredterra commented on banking/klarna_kosma_integration/doctype/banking_settings/banking_settings.py:
```diff
@@ -145,3 +145,28 @@ def get_customer_portal_url() -> str:
+	current_app_version = frappe.get_attr("banking.__version__")
+
+	latest_app_version = str(check_release_on_github("banking")[0])
+	if latest_app_version != current_app_version:
```
- Shouldn't this be `<`? (convert `current_version` to a `Version` object first)
- The latest release should probably be specific to the current branch. (A v15 release won't be of much use for a v14 user -- applies to ERPNext as well, btw)

marination replied:
> applies to ERPNext as well, btw

Do you suggest i do this check for erpnext too ?

barredterra replied:
If it's easier to fix there, it wouldn't hurt :D

marination replied:
An issue could be due to frappe being outdated too. Let's stick to banking. I think we're entering devops territory 😄

marination committed:
fix: get latest release for current branch & use Version for comparison

- Use github api and fetch latest release of current branch
- Use Version to compare

marination committed:
Merge branch 'version-14' into app-health
